README.md

```diff
@@ -2,7 +2,7 @@
 
 A GitHub Repository to encourage and involve beginners in Open Source Contributions
 
-Steps to follow:
+There are four Steps to follow:
```

Comment:
I think this is likely spam based on reading [this](: https://blog.domenic.me/hacktoberfest/) and seeing similar patterns of this user's sudden prs with no description context, etc in repos that I maintain. This makes the documentation harder to maintain. Adding a new item you should not have to think about updating multiple locations.

Reply:
@majormoses Thanks for the info. Will tag this as `spam`.